### PR TITLE
Allow for ignoring devDependencies via boolean config option ignoreDevDependencies

### PR DESCRIPTION
**Pull request by palmerj3**

### CODE REVIEW

**Comment by palmerj3** [comment]
@hzoo can you take a look?

**Comment by hzoo** [comment]
Oh yeah, took a break last week and forgot about this!

File: index.js [hunk 1]
```diff
@@ -47,7 +48,8 @@ module.exports = function checkLicenses(config) {
   }
 
   checker.init({
-    start: process.cwd()
+    start: process.cwd(),
+    production: ignoreDevDependencies
```
Comment (hzoo) [inline index.js]: Cool `--production only show production dependencies.` https://github.com/davglass/license-checker#options

**Comment by hzoo** [comment]
If you want we can add an example output for the options

**Comment by hzoo** [comment]
Awesome thanks!

**Pull request merged**